sympy/physics/mechanics/functions.py

```diff
@@ -532,7 +539,7 @@ def _recurser(expr, sub_dict):
 
 def _fraction_decomp(expr):
     """Return num, den such that expr = num/den"""
-    if not isinstance(expr, Mul):
+    if not isinstance(expr, MulClass):
```

Comment:
Ditto for `tan` and `Tan`. This worries me. If the SymEngine wrappers don't use the same names for functions and their classes like SymPy does, then compatibility lines like this will be a PITA to support in SymPy.